client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1172,7 +1172,7 @@ Undocumented.prototype.fetchSiteRecommendations = function( query, fn ) {
 };
 
 Undocumented.prototype.readRecommendationsStart = function( query, fn ) {
```

Comment:
Might be worth us defining a new endpoint `Undocumented.prototype.readRecommendationsPosts` perhaps rather than altering the existing one, just in case we want to test old results vs ElasticSearch results in the future?